ert_storage/endpoints/records.py

```diff
@@ -561,6 +561,19 @@ def _get_and_assert_ensemble(
 
     q = db.query(ds.Record).filter_by(ensemble_pk=ensemble.pk, name=name)
     if realization_index is not None:
+        if realization_index >= ensemble.size:
```

Comment:
Also check that it's less than zero. 
```py
realization_index not in range(ensemble.size)
```

Reply:
> If we're doing validation, you also need a validation for the size being out of range.
> 
> Although, it seems like this PR has been going on for a while and we want it in. I propose we postpone the validation to another PR and get this in sans validation, so as to get ERT2&3 on board.

I don't fully understand what you mean by validation for the size being out of range, but at least i introduced the not in range part which looks nicer. I also added -1 as an exception from the rule for now.